dipy/workflows/tests/test_io.py

```diff
@@ -51,6 +52,11 @@ def test_io_fetch():
                                                     'bundle_fa_hcp')),
                          True)
 
+        fetch_flow.run(['all'])
```

Comment:
Just remove this block, I do not think we can test it without a `TimeOut` error

Reply:
If we remove it, the error is likely to appear at some point in the future. I do acknowledged that I have no workaround for the `TimeOut` at the moment. But the culprit dataset is `bundle_fa_hcp`, which is fetched in the two tests before, so this is striking.

Reply:
I do not think this is the only one "culprit" dataset, specially when you look at each CI's. Sometimes multi_cenir, other times  multi_cfin, etc...
In general, I do not think it is a good idea to download heavy data during unittest
Unfortunately, `fetch_all` can not be tested  (fetching part) but if you really want a workaround, you can create a static method in this class named get_fetcher_datanames and compare the results of this function what we expect. This function will replace the line 158.

Reply:
That looks reasonable. @arokem @Garyfallidis any other suggestion?

Reply:
Yeah. I think that's reasonable. If we figure out a way to continuously build/deploy the docs, the fetching operation itself will presumably get tested there.

Reply:
@skoudoro @arokem Please have a look at commit 082c1c0.